setup.cfg

```diff
@@ -2,33 +2,75 @@
 python-tag = py34.py35.py36
 
 [tool:pytest]
-norecursedirs =
-    .git
-    dist
-    build
-    venv
-    .env
-    .vendor
-    vendor
-    tests/python/bandit_test_files
-    tests/python/pyroma_test_files
+minversion = 3.6.1
+
+testpaths =
+  bears
+  tests
+
 python_files = *Test.py
 python_classes = *Test
-timeout = 35
+
 addopts =
-    --color=yes
-    --ignore=test-requirements.txt
-    --profile
+  -r a
+  -p no:logging
+  --color=yes
+  --doctest-glob=*.rst
+  --doctest-modules
+  --doctest-ignore-import-error
+  --cov
+  --instafail
+  --profile
+  --reorder 'requirements.txt' 'test-requirements.txt' '*'
+  --ignore=tests/python/bandit_test_files/
+
+doctest_optionflags =
+  ELLIPSIS
+  IGNORE_EXCEPTION_DETAIL
+
 env =
-    PYTHONHASHSEED=0
-# PYTHONHASHSEED=0 is required to use same hashes in pytests-xdist's workers
+  PYTEST=1
+
 python_paths = .ci/
 
+reqsfilenamepatterns =
+  requirements.txt
+  test-requirements.txt
+
+reqsignorevcs = True
+
+timeout = 35
```

Comment:
appveyor is complaining about timeout problem... Maybe also increase the timeout value in this PR?

Reply:
That commit isnt part of this PR.

I saw the problem once or twice, at least once in DocGrammarBear.  Needs a separate issue.  As that is a native bear, it should be quick -- if it isnt likely the problem is in the bear logic.